lib/active_model_serializers/key_transform.rb

```diff
@@ -11,6 +11,7 @@ module KeyTransform
     # @see {https://github.com/rails/rails/blob/master/activesupport/lib/active_support/inflector/methods.rb#L66-L76 ActiveSupport::Inflector.camelize}
     def camel(value)
       case value
+      when Array then value.map { |item| camel(item) }
```

Comment:
This is a separate feature that should be noted in the changelog, I think. Otherwise, LGTM